ludwig/models/llm.py

```diff
@@ -221,12 +221,36 @@ def initialize_adapter(self):
                     "`finetune` or remove the adapter config."
                 )
 
-            from peft import get_peft_model, TaskType
+            from peft import get_peft_model
+
+            pretrained = False
+            if self.config_obj.adapter.pretrained_weights:
+                print(f"PRETRAINED_WEIGHTS: {self.config_obj.adapter.pretrained_weights}")
```

Comment:
nit: Are we leaving this `print` in? If so, could you change it to `logger.info`?